guides/main_concepts/documents.md

```diff
@@ -60,26 +66,24 @@ In an index called `movie` there are 200k `documents`. Each of these 200k docume
 ]
 ```
 
-If the **primary key** is not found in a document, the document will not be added.
-
 ### Setting the primary key
 
-MeiliSearch has several ways of knowing which field is the `primary key`.
+MeiliSearch can deduce which field is the `primary key` in several ways.
```

Comment:
Is it deducing when you give the information using methods 2 and 3?

Reply:
Yes, you're right! What about:
```suggestion
There are several ways for MeiliSearch to know which field is the `primary key` in several ways.
```

Reply:
yes! Thanks!